src/components/traces/traceDetailsRow.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Expedia, Inc.
+ *
+ *       Licensed under the Apache License, Version 2.0 (the "License");
+ *       you may not use this file except in compliance with the License.
+ *       You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *       Unless required by applicable law or agreed to in writing, software
+ *       distributed under the License is distributed on an "AS IS" BASIS,
+ *       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *       See the License for the specific language governing permissions and
+ *       limitations under the License.
+ *
+ */
+
+/* eslint-disable react/prefer-stateless-function */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class TraceDetailsRow extends React.Component {
```

Comment:
A better name could just be SpanRow or just Span, since a row corresponds to a single span